src/lib/serviceRoutinesV2/getEntityAttribute.cpp

```diff
@@ -74,7 +74,8 @@ std::string getEntityAttribute
 
   // 01. Fill in QueryContextRequest
   parseDataP->qcr.res.fill(compV[2], type, "false", EntityTypeEmptyOrNotEmpty, "");
-
+  parseDataP->qcr.res.metadataList.fill(ciP->uriParam["metadata"]);
```

Comment:
A define should be used instead of `"metadata"`, as done with URI_PARAM_ATTRIBUTES.